tests/in/glsl/bevy-pbr.frag

```diff
@@ -0,0 +1,368 @@
+// NOTE: Taken from the bevy repo
```

Comment:
Just to be sure but we have permission to use this shader in our test corpus right? Maybe adding some kind of license header or something maybe be needed

Reply:
I've asked about that and we concluded that the safest choice is to include the bevy MIT license verbatim in the file header.